cms/envs/common.py

```diff
@@ -1468,6 +1468,7 @@
     'openedx.features.discounts',
     'experiments',
 
+    'openedx.core.djangoapps.external_user_ids',
```

Comment:
Are we using the external ID in Studion now? This was not added on purpose.

If we are not using external IDs in Studio we should not add it as an app to the service.  There is an annotation to have tests only run for the LMS.

Here is an example: https://github.com/edx/edx-platform/blob/7a52a251324a851b9aa6f135aee8cc1dee366500/openedx/core/djangoapps/external_user_ids/tests/test_signals.py#L33

Reply:
The app is added to studio so that LTI launches from studio will be able to create external user ids.

Reply:
Do you mind removing the `skip_unless_lms` annotation from the external_user_ids tests in this PR as well then?

Now that it is in Studio